pkg/virt-operator/resource/apply/core_test.go

```diff
@@ -53,7 +53,7 @@ var _ = Describe("Apply", func() {
 
 	Context("Services", func() {
 
-		It("should patch if ClusterIp == \"\" during update", func() {
+		It(`should not patch if ClusterIp == "" during update`, func() {
```

Comment:
Maybe it'd be better phrase it something like `should not patch if ClusterIp is empty during update`?

Reply:
done